TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -499,75 +504,14 @@ public class RoutingManager
-                    prevSegmentId,
-                    prevLaneIndex,
-                    prevLaneId,
-                    isNodeStartNodeOfPrevSegment,
-                    prevSegmentId,
-                    nextSegmentId,
-                    nodeId,
-                    onHighway,
-                    applyHighwayRules,
-                    applyHighwayRulesAtJunction,
-                    Options.highwayRules,
-                    nodeIsSimpleJunction,
-                    nextAreOnlyOneWayHighways,
-                    prevEnd.outgoing && prevExtSegment.oneWay,
-                    prevExtSegment.highway,
-                    iterateViaGeometry);
-                Log._DebugFormat(
-                    "RoutingManager.RecalculateLaneEndRoutingData({0}, {1}, {2}, {3}): " +
-                    "prevSegIsInverted={4} leftHandDrive={5}",
-                    prevSegmentId,
-                    prevLaneIndex,
-                    prevLaneId,
-                    isNodeStartNodeOfPrevSegment,
-                    prevSegIsInverted,
-                    Shortcuts.LHT);
-                Log._DebugFormat(
-                    "RoutingManager.RecalculateLaneEndRoutingData({0}, {1}, {2}, {3}): " +
-                    "prevSimilarLaneCount={4} prevInnerSimilarLaneIndex={5} prevOuterSimilarLaneIndex={6} " +
-                    "prevHasBusLane={7}",
-                    prevSegmentId,
-                    prevLaneIndex,
-                    prevLaneId,
-                    isNodeStartNodeOfPrevSegment,
-                    prevSimilarLaneCount,
-                    prevInnerSimilarLaneIndex,
-                    prevOuterSimilarLaneIndex,
-                    prevHasBusLane);
-                Log._DebugFormat(
-                    "RoutingManager.RecalculateLaneEndRoutingData({0}, {1}, {2}, {3}): nextIsJunction={4} " +
-                    "nextIsEndOrOneWayOut={5} nextHasTrafficLights={6} nextIsSimpleJunction={7} " +
-                    "nextIsSplitJunction={8} isNextRealJunction={9}",
-                    prevSegmentId,
-                    prevLaneIndex,
-                    prevLaneId,
-                    isNodeStartNodeOfPrevSegment,
-                    nodeIsJunction,
-                    nodeIsEndOrOneWayOut,
-                    nodeHasTrafficLights,
-                    nodeIsSimpleJunction,
-                    nodeIsSplitJunction,
-                    nodeIsRealJunction);
-                Log._DebugFormat(
-                    "RoutingManager.RecalculateLaneEndRoutingData({0}, {1}, {2}, {3}): nextNodeId={4} " +
-                    "buildingId={5} isTollBooth={6}",
-                    prevSegmentId,
-                    prevLaneIndex,
-                    prevLaneId,
-                    isNodeStartNodeOfPrevSegment,
-                    nodeId,
-                    buildingId,
-                    isTollBooth);
-            }
+            extendedLog?.Invoke(new { prevSegmentId, _ = "Starting exploration with ",
```

Comment:
did you use any automated process to generate output? Maybe instead of checking if every single variable I can check your process.

Reply:
I hardcoded `extendedLogRouting` to `true` and spent a fair bit of time scrutinizing what came out.